Marlin/src/module/configuration_store.cpp

```diff
@@ -1835,11 +1833,12 @@ void MarlinSettings::reset(PORTARG_SOLO) {
     #endif
   #endif
 
+
+  z_raise = TOOLCHANGE_ZRAISE;
```

Comment:
Please don't add random blank lines. You should see the amount of cleanup I already did on this PR.